Azure/communication-ui-library-ios
# [Feature] Redux layer and Presentation layer for receive read receipt

## Purpose

* When joining or re-joining a chat thread, the last send message will show read icon
* The last message that is received by all participants will show read icon

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test

* Enter a chat
* Send a message
* Make sure message is seen by all the other participants in this chat


## What to Check

* When open an existing chat which has messages, the last sent message should show read icon
* The last message that is read by all other participants should show read icon

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

<img src="https://user-images.githubusercontent.com/107075081/201773129-31ed70aa-ef06-418f-88a7-6da8b7980c88.png" width="200" height="400" /> <img src="https://user-images.githubusercontent.com/107075081/201773238-2e9ed026-ddda-43c9-8b23-9d41339ff1a2.png" width="200" height="400" />

## Other Information

N/A
